File: C++/Problems/Problems/problem_319.cpp
#include "problem_319.h"

bool problem_319::test() {
    int n = 3;

    int expected = 1;

    auto result = bulbSwitch(n);

    std::cout << result << std::endl;

    return result == expected;
}

int problem_319::bulbSwitch(int n) {
    return (int)std::sqrt(n);
}

File: Python/Problems/Problems/problem319.py
import heapq
import math
import sys

from utils import print1DArray, print2DArray
from typing import List, Set, Optional
from collections import Counter

from .problem import Problem

class Problem319(Problem):
    def test(self) -> bool:
        n = 3

        expected = 1

        result = self.bulbSwitch(n)

        print(result)

        return result == expected

    def bulbSwitch(self, n: int) -> int:
        return int(math.sqrt(n))